API.md

```diff
@@ -0,0 +1,1819 @@
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "1",
+            "text": "difficulty-breathing",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "2",
+            "text": "fever",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "3",
+            "text": "used-a-fever-reducer",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "4",
+            "text": "chills",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "5",
+            "text": "repeated-shaking-with-chills",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          },
+          {
+            "linkId": "6",
+            "text": "muscle-pain",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "7",
+            "text": "headache",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "8",
+            "text": "sore-throat",
+            "answer": [
+              {
+                "valueBoolean": false
+              }
+            ]
+          },
+          {
+            "linkId": "9",
+            "text": "new-loss-of-taste-or-smell",
+            "answer": [
+              {
+                "valueBoolean": true
+              }
+            ]
+          }
+        ],
+        "resourceType": "QuestionnaireResponse"
+      }
+    }
+  ],
+  "resourceType": "Bundle"
+}
+```
+</details>
+
+
+<a name="create"/>
+
+## Creating
+
+The API supports creating new monitorees.
+
+### Extensions
+
+<a name="create-ext"/>
+
+Along with supporting the US Core extensions for race, ethnicity, and birthsex, Sara Alert includes four additional extensions for things specific to the Sara Alert workflows.
+
+Use `http://saraalert.org/preferred_contact_method` for specifying the monitorees Sara Alert preferred contact method (options are: `E-mailed Web Link`, `SMS Texted Weblink`, `Telephone call`, and `SMS Text-message`).
+
+```json
+{
+  "url": "http://saraalert.org/preferred_contact_method",
+  "valueString": "E-mailed Web Link"
+}
+```
+
+Use `http://saraalert.org/symptom_onset_date` to specify when the monitoree's first symptoms appeared for use in the Sara Alert isolation workflow.
+
+```json
+{
+  "url": "http://saraalert.org/symptom_onset_date",
+  "valueDate": "2020-05-23"
+}
```

Comment:
This should probably be an Observation, "Date of Onset - Reported" https://loinc.org/76425-8/ or similar